copyables/entrypoint.sh

```
@@ -156,14 +156,18 @@ if [ ! -f $CONFIG ] || [ ! -s $CONFIG ]; then
 
   # handle VPNCMD_* commands right before setting admin passwords
   if [[ $VPNCMD_SERVER ]]; then
-    while IFS=";" read -ra CMD; do
+    while IFS=";" read -r -a CMD; do
+      set -f
       vpncmd_server $CMD
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)